github-orgs/kubeflow/org.yaml

```diff
@@ -5,12 +5,16 @@
 orgs:
     kubeflow:
         admins:
+        - andreyvelich
         - caniszczyk
         - chensun
         - google-oss-robot
         - googlebot
         - james-jwu
+        - jbottum
+        - johnugeorge
         - krook
+        - terrytangyuan
         - theadactyl
```

Comment:
We can remove Thea since she’s no longer involved

Reply:
Done.